### PR TITLE
Updating ciceroscmpy version and tests this also gets rid of pandas <…

### PR DESCRIPTION
**Pull request by maritsandstad**
…2 requirement

### CODE REVIEW

**Comment by maritsandstad** [comment]
Should I try to up the test coverage for my various help functions in this while I'm at it @znicholls ?

(Otherwise this might just be a quick update version thing just to get less outdated versioning...)

**Comment by znicholls** [comment]
> Should I try to up the test coverage for my various help functions in this while I'm at it @znicholls ?

If you want, can also be done in a follow up PR to keep the scope of each smaller

**Comment by maritsandstad** [comment]
Seems I need to spend some time fighting with windows first, anyways...

**Comment by znicholls** [comment]
> Seems I need to spend some time fighting with windows first, anyways...

The error is really bizarre. The CI shows the environment at the top in case that's helpful (maybe different package versions available on windows compared to linux...)

e.g. CI https://github.com/openscm/openscm-runner/actions/runs/7708984548/job/21009201207?pr=89

If you scroll to the right bit you see

Screen Shot 2024-01-30 at 11 39 01 am

**Comment by maritsandstad** [comment]
Ok, fixed the windows thing, apparently windows default int type for numpy is np.int32... Anyways, the fail now is coverage, so if the tests are supposed to go green, then I might need to work on that...

File: pyproject.toml [hunk 1]
```diff
@@ -48,12 +48,12 @@ python-dotenv = "^1.0.0"
...
 tqdm = "^4.66.1"
 # optional below here
-ciceroscm = { version = "1.0.0", optional = true }  # TODO: ask Marit if she wants to loosen this
+ciceroscm = { version = "1.1.1", optional = true }  # TODO: ask Marit if she wants to loosen this
 fair = { version = "<2", optional = true }
 pymagicc = { version = "<3", optional = true }
 # Bit hacky, but basically we only need to pin pandas
 # if we're installing ciceroscm
-pandas = { version = "<2", optional = true }
+pandas = { version = "*", optional = true }
```
Comment (znicholls) [inline pyproject.toml:56]: We can just delete all these lines now I think

File: pyproject.toml [hunk 2]
```diff
@@ -48,12 +48,12 @@ python-dotenv = "^1.0.0"
 scmdata = ">=0.7.4"
 tqdm = "^4.66.1"
 # optional below here
-ciceroscm = { version = "1.0.0", optional = true }  # TODO: ask Marit if she wants to loosen this
+ciceroscm = { version = "1.1.1", optional = true }  # TODO: ask Marit if she wants to loosen this
```
Comment (znicholls) [inline pyproject.toml:51]:
```suggestion
ciceroscm = { version = "1.1.1", optional = true }
```

**Review by znicholls — APPROVED** [review]
Don't worry about the coverage, that's just coverage being silly (I'll just override it when I merge).

If you can mark the PR as ready for review and make the updates then I'll merge assuming everything goes green (once it's no longer a draft, a couple more things get run which may flag further errors).

**Pull request merged**